_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+        <legend class="group-legend is-hidden">Personal Details</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="full_name_default">Your name (default text field)</label>
+            <input class="input input-text" type="text" name="full_name_default" id="full_name_default" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="email_address_default">Email address (default, disabled text field)</label>
+            <input class="input input-text" type="email" name="email_address_default" id="email_address_default" placeholder="e.g. jane@email.com" disabled>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label required-field" for="email_address_default_helper">Email address (default, required text field with helper text)</label>
+            <input class="input input-text" type="email" name="email_address_default_helper" id="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper" required>
+            <div class="helper-text" id="email_address_default_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="full_name_simple">Your name (simple text field)</label>
+            <input class="input input-text input-simple" type="text" name="full_name_simple" id="full_name_simple" placeholder="e.g. Jane Doe" required>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Select/Dropdown/Combo</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Some select menus</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="select_01">Default select menu</label>
+            <select class="input input-select" name="select_01" id="select_01">
+                <option value="">- Choose one -</option>
+                <option value="Mary">Mary</option>
+                <option value="had">had</option>
+                <option value="a">a</option>
+                <option value="Little">Little</option>
+                <option value="Lamb">Lamb</option>
+            </select>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="select_01">Enhanced select menu</label>
+            <select class="input input-select replace-select" name="select_02" id="select_02">
+                <option value="">- Choose one -</option>
+                <option value="Mary">Mary</option>
+                <option value="had">had</option>
+                <option value="a">a</option>
+                <option value="Little">Little</option>
+                <option value="Lamb">Lamb</option>
+            </select>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Radios</h4>
```

Comment:
Add ``.example-set-hd`` class in here for consistency and extra PL Doc site formatting.